core/opendaq/device/include/opendaq/device_impl.h

```diff
@@ -183,10 +186,12 @@ GenericDevice<TInterface, Interfaces...>::GenericDevice(const ContextPtr& ctx,
 {
     this->defaultComponents.insert("Dev");
     this->defaultComponents.insert("IO");
+    this->defaultComponents.insert("Synchronization");
```

Comment:
Could we maybe try if things work properly over OPC UA if we rename this to "Sync"?